fabtests/common/shared.c

```diff
@@ -1427,6 +1427,14 @@ int ft_enable_ep(struct fid_ep *bind_ep, struct fid_eq *bind_eq, struct fid_av *
 		FT_EP_BIND(bind_ep, bind_rma_cntr, flags);
 	}
 
+	if (opts.max_msg_size) {
+		ret = fi_setopt(&bind_ep->fid, FI_OPT_ENDPOINT, FI_OPT_MAX_MSG_SIZE, &opts.max_msg_size, sizeof opts.max_msg_size);
+		if (ret && ret != -FI_ENOPROTOOPT) {
```

Comment:
I think it's a bug in the man page. In this case we should return `-FI_EOPNOTSUPP`

Reply:
Nvm, I think FI_ENOPROTOOPT is OK and that's what efa and shm returns today